apps/src/templates/sectionProgress/progressTables/ProgressTableLessonNumber.jsx

```diff
@@ -90,8 +94,12 @@ class ProgressTableLessonNumber extends React.Component {
       >
         {this.renderTooltip()}
         {lockable ? <FontAwesome icon="lock" /> : number}
-        {includeArrow && <span style={styles.line} />}
-        {includeArrow && <span style={styles.arrow} />}
+        {includeArrow && (
```

Comment:
I thought it was nice to have the arrow encapsulated in an element, but it's not necessary.